tests/semanticdb/metac.expect

```diff
@@ -1279,14 +1288,14 @@ Symbols => 8 entries
 Occurrences => 18 entries
 
 Symbols:
-ext/Extension$package. => final package object ext
-ext/Extension$package.`#*#`(). => method #*#
-ext/Extension$package.`#*#`().(i) => param i
-ext/Extension$package.`#*#`().(s) => param s
-ext/Extension$package.a. => val method a
-ext/Extension$package.c. => val method c
-ext/Extension$package.foo(). => method foo
-ext/Extension$package.foo().(s) => param s
+ext/Extension$package. => final package object ext extends Object { self: <?>.ext.type => +5 decls }
+ext/Extension$package.`#*#`(). => method #*# (param s: String)(param i: Int): <?>
```

Comment:
it seems to be a common occurrence for extension methods with two parameter lists to have an empty return type.

Reply:
see my other comment, it is probably all curried functions acting this way

Reply:
139491b